staging/src/k8s.io/apiserver/pkg/util/webhook/webhook_test.go

```diff
@@ -688,7 +687,7 @@ func TestWithExponentialBackoff(t *testing.T) {
 }
 
 func bootstrapTestDir(t *testing.T) string {
-	dir, err := ioutil.TempDir("", "")
+	dir, err := os.MkdirTemp("", "")
```

Comment:
Is there a `utiltesting.RemoveTestFile(t, dir)` needed here too?

Reply:
> Is there a `utiltesting.RemoveTestFile(t, dir)` needed here too?

dir seem has been already recycled in the upper function